conf/app.routes

```diff
@@ -1,6 +1,6 @@
 # JSON Based
 POST        /declarations                                           uk.gov.hmrc.exports.controllers.DeclarationController.create
-GET         /declarations                                           uk.gov.hmrc.exports.controllers.DeclarationController.findAll(status: Seq[String] ?= Seq.empty, page: Page ?= Page(), sort: DeclarationSort ?= DeclarationSort())
+GET         /declarations                                           uk.gov.hmrc.exports.controllers.DeclarationController.fetchPage(status: Seq[String] ?= Seq.empty, page: Page ?= Page(), sort: DeclarationSort ?= DeclarationSort())
```

Comment:
Would it not be safer to define a new endpoint url that has the new behavior and leave the existing endpoint as is (to be cleaned up later)?
This way we can deploy the BE first with no impact on the FE, then deploy the FE afterwards. At the moment which ever we deploy first FE or BE will cause the service to error (especially as we now have 14 instants to deploy to in prod!).